doc/ideas/gameplay.md

```diff
@@ -152,16 +257,90 @@ Multiplayer
+
+
+ - ZOOM! (also in-game not just for casters)
+ - maybe even turning the game in 90° angles
+   - the old models could stay 2D and just be fit into the turned environment(?)
+
+ - the feature set of CaptureAge could be understood as a spectator mode
+   - if they add Picture-in-Picture and replay-functions goes more into direction what's understood in this context as casting mode
+
+ - casting mode should help you to either work on your own or collaboratively work together with one or more Co-Caster(s) on the presentation of the actions in a game
+
+   - Casting layer: For casters and co casters. Could support features like pinging something on the map for the co caster, marking points of interest/units/buildings, show picture-in-picture views of the other casters and so on. An important requirement would be that the casters' views are synced. (_ColonelPanic_ @reddit)
+   - casters should have the possibility to join a live-game together as a caster-team
+   - e.g. two separate openage-accounts could be temporarily bound together (master-slave-principle)
+     - they both work together at the master-device (streaming device from main caster), each caster stil has it's own independant view of the game
+
+   - there should be an option to cut scenes together in a live game, so that if one caster sees an action somewhere on the map, he/she gives a command to the game that itself clips (saves camera position and game status) it and puts it into a queue on the master-device to replay it
+
+   - outside of the main spectating window could be a second window (used for two monitor solutions) with the waiting list for the replays and all the other options and features special to the casting mode
+
+ - in-game lag-free voice-communication between casters
+
+ - actions per minute (APM) should be derivable from game -> possible important influence on design decisions!
+
+ - show current camera locations of casters (maybe on minimap)
+
+ - small indicators for units that show if they are visible to another player, even if the FOW is turned off
+   - to get rid of the switching between the FOW of players just to know, if they can see each other
+   - see also: Viewcone-Feature (Unit handling)
+
+
 ### Team interaction
 
  - Better team interaction
  - Work/resource sharing
  - Unit sharing (transport boats)
+  - give temporary unit control of a selected group to an ally (e.g. in a fight) with an easy command
  - Give resources of resigning players to allies
+  - maybe even the research of team ressources in late-imp
+  - or could be even a 5th "basic globalisation/Hanse/free trade" age where allys share ressources
+  - when you resign, give your buildings to your allies (original AoE devs idea)
  - Color markings
  - Create signs
  - Paint on map
  - Create arrows
+ - in-game lag-free voice-communication between teamplayers
```

Comment:
This is already mentioned somewhere else

Reply:
"- in-game lag-free voice communication between casters"
This is the other mention, here it is about casters. Think it is worth to mention, that it's not the same level of voice communication and they both exists in parallel.